### ISSUE TITLE
Allow SQL to be passed 'preformatted'

### ISSUE THREAD

**Issue by fhoeben** [six42/jdbcslim#11]
When SQL statements become more complex it is nice to format them in the wiki (so they can be across multiple lines and can have some indentation).
At the moment we cannot pass such SQL to SQLCommand, as it gets the HTML formatted version and tries to pass that to the JDBC driver, which (suprise! :-)) does not understand SQL surrounded between `<pre>` and `</pre>` tags.

SQLCommand could check whether the SQL starts and ends with these tags, and remove them if present before passing to the JDBC driver.

Sample usage based on part of PlugIns.JdbcSlim.XxTestMoreDrivers.HtwoTest.Tests.7UsingSqlInScriptsAndScenarios

```
!define SQL { {{{
select Name
from TestData
where
  ID = '@id'
}}} }

|scenario|The Id              |id   |is Named|name|and Has Phone|phone|
|execute |${SQL}                                                      |
|show    |success                                                     |
|show    |rawResult                                                   |
|show    |resultSheet                                                 |
|check   |getColumnValueByName|NAME |@name                            |
|#                                                                    |
|execute |{{{
select *
 from TestData
  where ID = '@id'
}}}|
|show    |success                                                     |
|show    |rawResult                                                   |
|show    |resultSheet                                                 |
|check   |getColumnValueByName|Phone|@phone                           |

```

**Comment by six42** [comment]
Replace {{{ SQL }}} with !- SQL -! 
And it will work.

**Comment by fhoeben** [comment]
Does `!--!` also allow for indentation of the SQL command's lines, and correct handling of `<` and `>`? I believe it will just add the contents to the HTML of the page (causing whitespace to be collapsed).

What do you think of the pull request?

**Comment by fhoeben** [comment]
Closed via #13
